sindresorhus/refined-github
# Add full names to events on for dashboard (extends `show-name`)

# Description
Show full names in dashboard.

![image](https://user-images.githubusercontent.com/55841/61146679-e39c2e00-a4da-11e9-86d7-17fd93158094.png)

# Closes
Closes https://github.com/sindresorhus/refined-github/issues/1065

# Test
* Go to your dashboard on https://github.com
* See full names being loaded on the dashboard feed.
* Scroll to the bottom, to load more.
* See full names being loaded on the dashboard feed again.

# Conflicting
This will conflict with https://github.com/sindresorhus/refined-github/pull/2193 but everything can be solved 😄
